alacritty/src/input/mod.rs

```diff
@@ -646,21 +647,21 @@ impl<T: EventListener, A: ActionContext<T>> Processor<T, A> {
                 self.ctx.clear_selection();
 
                 // Start new empty selection.
-                if self.ctx.modifiers().state().control_key() {
+                if control {
                     self.ctx.start_selection(SelectionType::Block, point, side);
                 } else {
                     self.ctx.start_selection(SelectionType::Simple, point, side);
                 }
             },
-            ClickState::DoubleClick => {
+            ClickState::DoubleClick if !control => {
                 self.ctx.mouse_mut().block_hint_launcher = true;
                 self.ctx.start_selection(SelectionType::Semantic, point, side);
             },
-            ClickState::TripleClick => {
+            ClickState::TripleClick if !control => {
                 self.ctx.mouse_mut().block_hint_launcher = true;
                 self.ctx.start_selection(SelectionType::Lines, point, side);
             },
```

Comment:
During expansion we **always** use block mode when control is pressed. But here when control is pressed it seems like we just don't select at all? We should definitely be consistent in this regard.

I'm probably fine with both, though "nothing happening" might be more confusing to users than just always having a block selection.

Reply:
It does select though, just don't trigger the other selections, since we're already inside the block selection.